api/src/main/resources/workbench-api.yaml

```diff
@@ -7289,6 +7289,9 @@ definitions:
       src_gke_cluster:
```

Comment:
do we need the cluster anymore or can we remove it?

Reply:
We may not need that but I am thinking it might be helpful to manually identified users because that is guarantee 1 to 1 mapping to a user. And it may break production system if we don't handle rolling out terrraform change and code change here well. so I just keep here and in sync with Terraform what we have.